packages/axios-extension/src/abap/adt-catalog/generators/ui-service-generator.ts

```diff
@@ -21,6 +21,23 @@ export class UiServiceGenerator extends AdtService {
         this.bo = bo;
     }
 
+    /**
+     * Get the schema of the service binding.
+     *
+     * @returns The schema of the service binding.
+     */
+    public async getSchema(): Promise<any> {
+        const response = await this.get('/schema', {
+            headers: {
+                Accept: 'application/vnd.sap.adt.repository.generator.schema.v1+json'
+            },
+            params: {
+                referencedObject: this.bo.uri
+            }
+        });
+        return JSON.parse(response.data);
```

Comment:
try catch?

Reply:
I have a catch for errors from the calling generator, this is also similar to the other methods here for service gen. If it's ok I'll follow that pattern here?

Reply:
OK. Thanks